app/controllers/webhooks_controller.rb

```diff
@@ -9,9 +9,11 @@ def email
         return
       end
 
-      if p.from.address == 'postmaster@hmps.gsi.gov.uk'
-        logger.error "Sender ( postmaster@hmps.gsi.gov.uk ) detected. Skipping message.\n #{p.inspect}"
-        render text: 'Sender postmaster@hmps.gsi.gov.uk'
+      blacklist = %w(postmaster sendmaster webmaster no-reply noreply root)
```

Comment:
Didn't @ashb mention that `root` shouldn't be included, or did you two decide it was fine to?